packages/components/tests/dummy/app/templates/components/pagination.hbs

```diff
@@ -484,7 +484,6 @@
     @showLabels={{true}}
     @showTotalItems={{false}}
     @showSizeSelector={{false}}
```

Comment:
I think there is a difference between the code snippet and the "renders to:" code. can you check?

Reply:
Fixed it, thanks